### ISSUE TITLE
Missing instructions on how to authenticate in CLI

### ISSUE THREAD

**Issue by ZeRego** [lightdash/lightdash#2864]
It seems we removed a section on how to authentication from this page 
`https://docs.lightdash.com/get-started/setup-lightdash/lightdash-cli/#authentication`

[Slack thread](https://lightdash-community.slack.com/archives/C03MG2VTR08/p1659640036419139)
> I’m trying lightdash preview for the first time and I get an error about not being logged in. I’m not sure what’s going on here, but when I follow the provided link to the docs (https://docs.lightdash.com/get-started/setup-lightdash/lightdash-cli/#authentication) it doesn’t seem to include auth info.

**Comment by TuringLovesDeathMetal** [comment]
https://docs.lightdash.com/guides/how-to-create-dimensions#2-login-to-the-lightdash-cli

**Comment by TuringLovesDeathMetal** [comment]
^This is the link it needs to be updated with in the CLI error message!

**Comment by TuringLovesDeathMetal** [comment]
Can we also make the link to the docs have a whitespace between it and the previous line, and be in a different colour?

If I look at the error message right now, it blends right in and I wouldn't read it (and this has happened in user onboarding too!)
 
<img width="383" alt="image" src="https://user-images.githubusercontent.com/31848148/185601201-3d65c351-048e-425b-b23e-e207ce38757f.png">

So, something like this:

```
looks like you didn't ... blah blah

See https://docs.lightdash.com/guides/how-to-create-dimensions#2-login-to-the-lightdash-cli for help and examples.
```

**Comment by TuringLovesDeathMetal** [comment]
Fixed in #3037
